src/fen_util.rs

```diff
@@ -164,12 +279,51 @@ pub fn parse_piece_placement(
     Ok(())
 }
 
-/// Parse whose turn it is.
+/// Parses whose turn it is and updates the chessboard.
+///
+/// # Arguments
+///
+/// - `chessboard`: A mutable reference to the `Chessboard` struct to update whose turn it is.
+/// - `whose_turn`: A string representing whose turn it is ("w" for white, "b" for black).
+///
+/// # Example
+///
+/// ```
+/// use kn_o3::Chessboard;
+///
+/// let mut chessboard = Chessboard::empty();
+/// let whose_turn = "w";
+/// parse_whose_turn(&mut chessboard, whose_turn);
+/// assert_eq!(chessboard.white_turn, true);
+/// ```
+///
+/// The function parses the turn information from the FEN string and updates the chessboard
+/// to reflect whose turn it is. It expects "w" for white and "b" for black.
```

Comment:
It's a 1 line function. DOES IT REALLY NEED 20 DOC STRING LINES?!?

Reply:
It doesn't